Bottom: 166c70b8b08b1e51d579786d9035fc2b04842bf6
Top:    2afaecec940c0e63141f649436b2011e67f100dd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:09:34 +0000

Refresh of wip-on-quad-edge-play

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 98d03d6..8dd2b81 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -7,9 +7,9 @@
 
 #include "quad.h"
 #include "vertex3d.h"
+#include "appearance.h"
 #include "face3d.h"
 #include "edge3d.h"
-#include "appearance.h"
 #include "object3d.h"
 #include "polygon.h"
 #include "data.h"
